index.bs

```diff
@@ -2900,9 +2902,17 @@ of running the following steps:
 
 <h3 id="obtain-aggregatable-report-aggregation-service-payloads">Obtaining an aggregatable report's aggregation service payloads</h3>
 
-To <dfn>obtain the public key for encryption</dfn> given an [=aggregation coordinator=]
-<var ignore=''>aggregationCoordinator</var>, asynchronously return a user-agent-determined public key
-or an error in the event that the user agent failed to obtain the public key.
+To <dfn>obtain the public key for encryption</dfn> given an [=aggregation coordinator=] |aggregationCoordinator|:
+
+1. [=Assert=]: |aggregationCoordinator| is not an [=opaque origin=].
```

Comment:
Is this assertion necessary? Can we instead specify that `aggregation coordinator` is a non-opaque origin?